uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/shared/components/leave-request-popup-details-tab.component.spec.js

```diff
@@ -1288,6 +1288,84 @@ define([
       });
     });
 
+    describe('time and date inputs watchers', function () {
+      beforeEach(function () {
+        var absenceTypes = absenceTypeData.all().values;
+
+        compileComponent({
+          mode: 'create',
+          role: 'admin',
+          selectedAbsenceType: absenceTypes[0]
+        });
+        spyOn(controller, 'calculateBalanceChange').and.callThrough();
+      });
+
+      describe('when the calculation unit is "hours"', function () {
+        beforeEach(function () {
+          selectedAbsenceType.calculation_unit_name = 'hours';
+        });
+
+        describe('when from/to deductions values are set but not changed', function () {
+          beforeEach(function () {
+            // the amounts are 0, setting them to 0 again still fires the watcher
+            controller.uiOptions.times.from.amount = 0;
+            controller.uiOptions.times.to.amount = 0;
+
+            $rootScope.$digest();
+          });
+
+          it('does not call the balance change calculation', function () {
```

Comment:
Can it be something like `does not call the balance change calculation function` or `does not perform the balance change calculation`